tests/acceptance/features/coreApiAuthWebDav/webDavCOPYAuth.feature

```diff
@@ -55,7 +55,7 @@ Feature: COPY file/folder
       | /remote.php/dav/spaces/%spaceid%/PARENT/parent.txt |
     Then the HTTP status code of responses on all endpoints should be "401"
 
-  @issue-ocis-reva-14
+
```

Comment:
Should we keep reva issue tag?

Reply:
The issue was in `ocis-reva` repo and closed there. Do we need this tag anymore? @saw-jan

Reply:
No we don't. I thought reva issue